tests/Doctrine/Tests/DBAL/Functional/Schema/ComparatorTest.php

```diff
@@ -25,15 +25,31 @@ protected function setUp()
         $this->comparator    = new Comparator();
     }
 
-    public function testDefaultValueComparison()
+    /**
+     * @param mixed $value
+     *
+     * @dataProvider defaultValueProvider
+     */
+    public function testDefaultValueComparison(string $type, $value)
```

Comment:
` : void` could be added

Reply:
@Majkl578 we may have already discussed it, but is it possible for the coding standard to require/add `: void` if a function doesn't `return` or `yield` anything?

Reply:
It does (SlevomatCodingStandard.TypeHints.TypeHintDeclaration.MissingReturnTypeHint), but it's disabled in master.

Reply:
It should be safe to enable it for tests, right? Maybe except for mocks. I hate those mocks.

Reply:
> It should be safe to enable it for tests, right?

It should.